packages/browser-integration-tests/utils/fixtures.ts

```diff
@@ -23,9 +23,12 @@ const getAsset = (assetDir: string, asset: string): string => {
   return `utils/defaults/${asset}`;
 };
 
-export type TestFixtures = {
+export type SentryTestFixtures = {
   _autoSnapshotSuffix: void;
   testDir: string;
+  bundle: string;
+  isTracingCapableBundle: () => boolean;
```

Comment:
l: can we shorten this to e.g. `hasTracing` and `hasReplay`? or something like this, it is rather verbose now IMHO 😅

Reply:
yes I agree that it is verbose but hasTracing is extremely ambiguous in the context of a test